### PR TITLE
update controller doc examples

### LINKED ISSUE #154: Controller document examples
capturing comments from https://github.com/w3c/vc-jose-cose/pull/144/files 

So we can merge it.

- We need examples with identifiers that are URLs
- We need examples with identifiers that use JWK Thumbprint URIs

### PR DESCRIPTION
**Pull request by decentralgabe**
fix #154

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/301.html" title="Last updated on Sep 20, 2024, 8:50 PM UTC (65e5d84)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/301/274d755...65e5d84.html" title="Last updated on Sep 20, 2024, 8:50 PM UTC (65e5d84)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1010,48 +918,142 @@ <h2 id="cnf">cnf</h2>
                 <a data-cite="VC-DATA-MODEL-2.0#dfn-verify">verification</a> process.
             </p>
             <p>
-                Use of a proof-of-possession key provided by the Holder to the Issuer
-                to establish a cryptographic binding to the Holder in the Verifiable Credential
-                that is verifiable by the Verifier in the Verifiable Presentation is RECOMMENDED.
+                Use of a proof-of-possession key provided by the [=Holder=] to the [=Issuer=]
+                to establish a cryptographic binding to the Holder in the [=Verifiable Credential=]
```
Comment (TallTed) [inline index.html:922]:
```suggestion
                to establish a cryptographic binding to the [=Holder=] in the [=Verifiable Credential=]
```

File: index.html [hunk 2]
```diff
@@ -1136,7 +1138,7 @@ <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
             have no integrity protection.
         </p>
         <p class="advisement">
-            Verifiable Credentials and Verifiable Presentations are not
+            [=Verifiable Credentials=] and [=Verifiable Presentations=] are not
             required to be secured or integrity protected or to contain a
```
Comment (TallTed) [inline index.html:1142]:
```suggestion
            required to be secured or integrity protected nor to contain a
```

File: index.html [hunk 3]
```diff
@@ -1880,12 +1882,12 @@ <h3 id="alg-jose">Algorithm for Verifying a Credential or Presentation Secured w
...
             </li>
             <li>
-                <code>inputDocument</code>: the verifiable credential secured as a JWT [[RFC7519]]
+                <code>inputDocument</code>: the [=verifiable credential=] secured as a JWT [[RFC7519]]
             </li>
         </ul>
         <p>
-            Upon receipt of the verifiable credential or presentation secured as a JWT
-            [[RFC7519]], the holder or verifier follows this algorithm:
+            Upon receipt of the [=verifiable credential=] or [=presentation=] secured as a JWT
```
Comment (TallTed) [inline index.html:1889]:
```suggestion
            Upon receipt of the [=verifiable credential=] or [=verifiable presentation=] secured as a JWT
```

File: index.html [hunk 4]
```diff
@@ -1941,7 +1943,7 @@ <h3 id="alg-sd-jwt">Algorithm for Verifying a Credential or Presentation Secured
             </li>
         </ul>
         <p>
-            Upon receipt of the verifiable credential or presentation secured with
+            Upon receipt of the [=verifiable credential=] or [=presentation=] secured with
```
Comment (TallTed) [inline index.html:1946]:
```suggestion
            Upon receipt of the [=verifiable credential=] or [=verifiable presentation=] secured with
```

File: index.html [hunk 5]
```diff
@@ -1999,13 +2001,13 @@ <h3 id="alg-cose">Algorithm for Verifying a Credential or Presentation Secured w
...
             <li>
-                <code>inputDocument</code>: the verifiable credential or verifiable presentation
+                <code>inputDocument</code>: the [=verifiable credential=] or [=verifiable presentation=]
                 secured with [[[RFC9052]]]
             </li>
         </ul>
         <p>
-            Upon receipt of the verifiable credential or presentation secured with
-            [[RFC9052]], the holder or verifier follows this algorithm:
+            Upon receipt of the [=verifiable credential=] or [=presentation=] secured with
```
Comment (TallTed) [inline index.html:2009]:
```suggestion
            Upon receipt of the [=verifiable credential=] or [=verifiable presentation=] secured with
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Editorial, for consistency & clarity

**Review by selfissued — APPROVED** [review]

**Comment by decentralgabe** [comment]
Updating examples, nothing normative. Editorial approval. Changes and merge conflicts addressed. Merging.

**Pull request merged**